src/types.ts

```diff
@@ -389,24 +389,54 @@ export interface OpenSeaAsset extends Asset {
   transferFeePaymentToken: OpenSeaFungibleToken | null
 }
 
+/**
+ * Defines a LastSale type which contains details about last sale of the token
+ */
 export interface LastSale {
+
+  // The type of event
   eventType: string
+
+  // The auction type
   auctionType: string
+
+  // The total price of the sale in the payment
```

Comment:
just reminded me that this price is in the event's payment_token, which isn't being serialized here (we need to do that)

Reply:
`totalPrice` does get serialized [here](https://github.com/ProjectOpenSea/opensea-js/pull/31/files#diff-e1495d267619047a7cca5cfe8f692729R272)

Reply:
maybe the line moved - it links to ` auctionType: asset.last_sale.auction_type,` which serializes whether it was an english/dutch auction. By payment_token, i mean we need to call `tokenFromJSON` on the asset_event.payment_token, since total_price is in that payment token, and if you don't know if it's eth or some erc20 token, you don't really know the price at all